<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSave</class>
 <widget class="QDialog" name="DialogSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/png/png/Save-icon.png</normaloff>:/png/png/Save-icon.png</iconset>
  </property>
  <widget class="QPlainTextEdit" name="TextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: File will be saved to program directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter file name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SubRip (.srt)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MicroDVD (.sub, .txt)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MPlayer(.sub)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSave</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
